ee/codegen/src/__test__/__snapshots__/project.test.ts.snap

```diff
@@ -405,6 +420,9 @@ class MyToolCallingNode(ToolCallingNode):
     class Outputs(ToolCallingNode.Outputs):
         text: str
         chat_history: List[ChatMessage]
+
+    class NodeTrigger(ToolCallingNode.Trigger):
+        merge_behavior = MergeBehavior.AWAIT_ATTRIBUTES
```

Comment:
[blocking] we actually don't want to codegen outputs that are already expected to be default behavior. The ToolCallingNode already has this merge behavior for this merge behavior